models/operatorgroup.go

```diff
@@ -0,0 +1,45 @@
+package models
+
+import (
+	log "github.com/sirupsen/logrus"
+)
+
+// OperatorGroup represents a Kubernetes OLM OperatorGroup resource.
+type OperatorGroup struct {
+	Resource `yaml:",inline"`
+	Spec     OperatorGroupSpec
+}
+
+type OperatorGroupSpec struct {
+	TargetNamespaces []string `yaml:"targetNamespaces,omitempty"`
+}
+
+// NewOperatorGroup creates a new OperatorGroup object. The object's
+// metadata.name is set to the value of the "namespace" parameter.
+// If the "allnamespaces" is falsy, spec.targetNamespaces is set to include
```

Comment:
Nit:  comment here should match the capitalization of the parameter. Also, I would probably go with `false` instead of `falsy` because it's a boolean variable and can't really be anything other than `true` or `false`.